mllib/src/test/scala/org/apache/spark/ml/regression/AFTSurvivalRegressionSuite.scala

```diff
@@ -60,12 +60,14 @@ class AFTSurvivalRegressionSuite extends SparkFunSuite with MLlibTestSparkContex
     assert(aftr.getMaxIter === 100)
     assert(aftr.getTol === 1E-6)
     val model = aftr.fit(datasetUnivariate)
+      .setQuantileProbabilities(Array(0.1, 0.8))
```

Comment:
It is better to put setters before fit to make the estimator and the model it produces consistent.